scripts/makefile/protobuf.mk

```diff
@@ -0,0 +1,40 @@
+
+###############################################################################
+###                                Protobuf                                 ###
+###############################################################################
+
+protoVer=0.14.0
```

Comment:
this is duplicated in 3 make files. 

Maybe as future improvement we can have version defined in one separate file and then used in those make files ? 

I worry, in the future we will upgrade those versions and forgot to do it in all places.